Form/Type/SeoMetadataType.php

```diff
@@ -91,6 +91,7 @@ public function configureOptions(OptionsResolver $resolver)
             'data_class' => $this->dataClass,
             'translation_domain' => 'CmfSeoBundle',
             'required' => false,
+            'by_reference' => false,
```

Comment:
adding this makes the test fail. i checked in the sandbox, and with PHPCR this leads to "A detached document was found through a child relationship during cascading a persist operation:  Symfony\\Cmf\\Bundle\\SeoBundle\\Doctrine\\Phpcr\\SeoMetadata@0000000073c1c05d0000000089dcaddb"

i will make the form type aware of whether we use ORM or not.